src/Servers/Kestrel/shared/TransportMultiplexedConnection.Generated.cs

```diff
@@ -27,11 +27,11 @@ internal partial class TransportMultiplexedConnection : IFeatureCollection
         private void FastReset()
         {
             _currentIConnectionIdFeature = this;
-            _currentIConnectionTransportFeature = this;
```

Comment:
This was an impossible cast; however since they were `object?` fields the compiler didn't detect it.